src/usr/local/www/firewall_rules_edit.php

```diff
@@ -2168,7 +2168,7 @@ function setOptText(target, val) {
 	// fields are disabled
 	function disable_most(disable) {
 		var elementsToDisable = [
-			'interface', 'proto', 'icmptype\\[\\]', 'srcnot', 'srctype', 'src', 'srcmask', 'srcbebinport', 'srcbeginport_cust', 'srcendport',
+			'interface', 'proto', 'icmptype\\[\\]', 'srcnot', 'srctype', 'src', 'srcmask', 'srcbeginport', 'srcbeginport_cust', 'srcendport',
```

Comment:
?

Reply:
This is a bug and it should definitely be `srcbeginport`.